.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    env:
+      NPM_CONFIG_PROGRESS: "false"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 6
```

Comment:
@ogonkov I presume this step implicitly performs `npm install`. Would it be somehow possible to pass the `--no-audit` flag? I'm asking because of https://github.com/jashkenas/backbone/issues/4246#issuecomment-1013860308.

Reply:
I just migrate exist travis config. We can improve it in this PR, sure

Reply:
Good point, I realize I'm moving the goalposts a bit. It's not a hard condition in any case, the most important thing is that this works again.